internal/network/manifests_generator.go

```diff
@@ -340,3 +348,30 @@ func (m *ManifestsGenerator) AddDisableVmwareTunnelOffloading(ctx context.Contex
 	}
 	return nil
 }
+
+const redirectTelemeterStageManifest = `
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    telemeterClient:
+	  telemeterServerURL: https://infogw.api.stage.openshift.com
+`
+
+// Default Telemeter server is prod.
+// In case the cluster is created in stage env we need to redirct to Telemter-stage
+// Note: There is no Telemeter-integraion so we won't redirect and we will let it fail on authN when reaching the
+// 		 wrong Telemeter server. We will do the same for all non-cloud envs
+func (m *ManifestsGenerator) AddTelemeterManifest(ctx context.Context, log logrus.FieldLogger, c *common.Cluster) error {
+	if strings.Contains(m.Config.ServiceBaseURL, "stage") {
```

Comment:
I think that instead of spamming the prod Telemeter auth service it will be better to use a dummy URL instead.

Reply:
You mean in all other cases which are not stage/prod to create a ConfigMap that redirect the metrics to a dummy URL? (instead of just letting them fail on auth when reaching prod ?)

Reply:
I am not sure how the cluster will react to a dummy URL. Since I don't have a real way to test it until it reaches stage I rather keep the default behaviour for all envs that we can't control.

Reply:
You can install a cluster and create the above configmap with a dummy URL.

Reply:
Also, you can install from assisted-test-infra it should act just the same (add dummy URL)
The only thing you can't test until you get to staging is the logic you currently have.

Reply:
I will do it.

Reply:
`// Note: There is no Telemeter-integraion so we won't redirect and we will let it fail on authN when reaching the// 		 wrong Telemeter server. We will do the same for all non-cloud envs`
Please DONT! What if we run a scale test, will it DDOS prod telemetry?

Reply:
btw, it can be completely disabled:
https://docs.openshift.com/container-platform/4.7/support/remote_health_monitoring/opting-out-of-remote-health-reporting.html#insights-operator-new-pull-secret_opting-out-remote-health-reporting

Reply:
@romfreiman This was the default behaviour until today, nothing has changed - all scale tests until today had this exact behaviour, I just described it in the comment - this is not new code.
I am aware of https://docs.openshift.com/container-platform/4.7/support/remote_health_monitoring/opting-out-of-remote-health-reporting.html#insights-operator-new-pull-secret_opting-out-remote-health-reporting but this will require modifying manifests created by openshift-installer (not just adding new custom manifests) - I don't think this is the solution we seek.

Reply:
Anyway it is getting changed to @eranco74 suggestion (using dummy URL)